code-pushup.config.ts

```diff
@@ -72,88 +66,10 @@ const config: CoreConfig = {
       license: 'MIT',
       type: 'module',
     }),
-
-    await lighthousePlugin(
-      'https://github.com/code-pushup/cli?tab=readme-ov-file#code-pushup-cli/',
-      { chromeFlags: DEFAULT_FLAGS.concat(['--headless']) },
-    ),
   ],
 
   categories: [
-    {
-      slug: 'performance',
-      title: 'Performance',
-      refs: [lighthouseGroupRef('performance')],
-    },
-    {
-      slug: 'a11y',
-      title: 'Accessibility',
-      refs: [lighthouseGroupRef('accessibility')],
-    },
-    {
-      slug: 'best-practices',
-      title: 'Best Practices',
-      refs: [lighthouseGroupRef('best-practices')],
-    },
-    {
-      slug: 'seo',
-      title: 'SEO',
-      refs: [lighthouseGroupRef('seo')],
-    },
-    {
-      slug: 'bug-prevention',
-      title: 'Bug prevention',
-      description: 'Lint rules that find **potential bugs** in your code.',
-      refs: [{ type: 'group', plugin: 'eslint', slug: 'problems', weight: 1 }],
-    },
-    {
-      slug: 'code-style',
-      title: 'Code style',
-      description:
-        'Lint rules that promote **good practices** and consistency in your code.',
-      refs: [
-        { type: 'group', plugin: 'eslint', slug: 'suggestions', weight: 1 },
-      ],
-    },
-    {
-      slug: 'code-coverage',
-      title: 'Code coverage',
-      description: 'Measures how much of your code is **covered by tests**.',
-      refs: [
-        {
-          type: 'group',
-          plugin: 'coverage',
-          slug: 'coverage',
-          weight: 1,
-        },
-      ],
-    },
-    {
-      slug: 'security',
-      title: 'Security',
-      description: 'Finds known **vulnerabilities** in 3rd-party packages.',
-      refs: [
-        {
-          type: 'group',
-          plugin: 'js-packages',
-          slug: 'npm-audit',
-          weight: 1,
-        },
-      ],
-    },
-    {
-      slug: 'updates',
-      title: 'Updates',
-      description: 'Finds **outdated** 3rd-party packages.',
-      refs: [
-        {
-          type: 'group',
-          plugin: 'js-packages',
-          slug: 'npm-outdated',
-          weight: 1,
-        },
-      ],
-    },
+    ...coverageCategories,
```

Comment:
Why not add `coverageCoreConfigNx()` to `mergeConfigs` instead? Otherwise I don't know why you created the function.

Reply:
I have to remove the fn I guess or implement a version that also works for project level. I did not find a helper for that in the coverage plugin.

Reply:
"implemented" :)